client/src/main.ts

```diff
@@ -80,6 +80,8 @@ export async function convertMixin(src: string, opts: Partial<VuedcOptions> = {}
  * @returns Converted Script Setup syntax
  */
 export async function convertScript(src: string, opts: Partial<VuedcOptions> = {}) {
+  resetRegistry();
+  
```

Comment:
Remove trailing space

```suggestion
```

The linter is complaining about trailing space here.